lib/ansiblelint/formatters/__init__.py

```diff
@@ -1,8 +1,10 @@
 """Output formatters."""
 import os
 from pathlib import Path
+from typing import Union
 
 from ansiblelint.color import Color, colorize
+from ansiblelint.errors import MatchError
```

Comment:
Since it's not necessary in runtime:
```suggestion
if TYPE_CHECKING:
    from ansiblelint.errors import MatchError
```